UICatalog/Scenarios/ASCIICustomButton.cs

```diff
@@ -0,0 +1,313 @@
+﻿using System;
```

Comment:
Well I kept it because of his complexity. Yes, I also added unit test that proves it must work as is and it'is in the `BorderTests` on the `BorderStyle_And_DrawMarginFrame_Gets_Sets` test.